otter/log/formatters.py

```diff
@@ -237,6 +239,22 @@ def audit_log_formatter(eventDict, timestamp, hostname):
     return audit_log_params
 
 
+@singledispatch
+def serialize_to_jsonable(obj):
+    """
+    Serialize any object to a JSONable form
+    """
+    return None
+
+
+@serialize_to_jsonable.register(basestring)
+def serialize_str(s):
+    """
+    Serialize string
+    """
+    return s
```

Comment:
Could you register the identity function for all of lists, tuples, strings, dicts, ints, floats, and longs?

```
for t in list, tuple, basestring, int, float, long:
    serialize_to_jsonable.register(t, identity)
```

(fwiw, I prefer this to a `map` or a listcomp to emphasize that it has side-effects)